readme.md

```diff
@@ -54,218 +54,218 @@ See the Jupyter notebooks under [`examples/`](examples) for how to use `pymatviz
+|               [`ptable_heatmap_splits_plotly(2_vals_per_elem)`](assets/scripts/ptable_plotly/ptable_heatmap_splits_plotly.py)               |              [`ptable_heatmap_splits_plotly(3_vals_per_elem)`](assets/scripts/ptable_plotly/ptable_heatmap_splits_plotly.py)               |
+|                                      ![ptable-heatmap-splits-plotly-2][ptable-heatmap-splits-plotly-2]                                      |                                     ![ptable-heatmap-splits-plotly-3][ptable-heatmap-splits-plotly-3]                                      |
+
+[ptable-hists]: assets/svg/ptable-hists.svg
+[ptable-lines]: assets/svg/homo-nuclear-mace-medium.svg
+[ptable-scatters-parity]: assets/svg/ptable-scatters-parity.svg
+[ptable-scatters-parabola]: assets/svg/ptable-scatters-parabola.svg
+[ptable-heatmap-splits-2]: assets/svg/ptable-heatmap-splits-2.svg
+[ptable-heatmap-splits-3]: assets/svg/ptable-heatmap-splits-3.svg
+[ptable-heatmap-splits-plotly-2]: assets/svg/ptable-heatmap-splits-plotly-2.svg
+[ptable-heatmap-splits-plotly-3]: assets/svg/ptable-heatmap-splits-plotly-3.svg
+[ptable-hists-plotly]: assets/svg/ptable-hists-plotly.svg
 
-## Phonons
+### Dash app using `ptable_heatmap_plotly()`
 
-See [`examples/mlff_phonons.ipynb`](https://github.com/janosh/pymatviz/blob/main/examples/mlff_phonons.ipynb) for usage example.
+See [`examples/mprester_ptable.ipynb`](https://github.com/janosh/pymatviz/blob/main/examples/mprester_ptable.ipynb).
 
-|           [`phonon_bands(bands_dict)`](pymatviz/phonons.py)           |             [`phonon_dos(doses_dict)`](pymatviz/phonons.py)             |
-| :-------------------------------------------------------------------: | :---------------------------------------------------------------------: |
-|                            ![phonon-bands]                            |                              ![phonon-dos]                              |
-| [`phonon_bands_and_dos(bands_dict, doses_dict)`](pymatviz/phonons.py) | [`phonon_bands_and_dos(single_bands, single_dos)`](pymatviz/phonons.py) |
-|                    ![phonon-bands-and-dos-mp-2758]                    |                    ![phonon-bands-and-dos-mp-23907]                     |
+[https://user-images.githubusercontent.com/30958850/181644052-b330f0a2-70fc-451c-8230-20d45d3af72f.mp4](https://user-images.githubusercontent.com/30958850/181644052-b330f0a2-70fc-451c-8230-20d45d3af72f.mp4)
 
-[phonon-bands]: assets/phonon-bands-mp-2758.svg
-[phonon-dos]: assets/phonon-dos-mp-2758.svg
-[phonon-bands-and-dos-mp-2758]: assets/phonon-bands-and-dos-mp-2758.svg
-[phonon-bands-and-dos-mp-23907]: assets/phonon-bands-and-dos-mp-23907.svg
+## Phonons
 
-### Dash app using `ptable_heatmap_plotly()`
```

Comment:
Looks like this has been put at the wrong position ef58e9e41cde30358e2d28e70e4b0ff2da82c41c